UI/sign-in/src/CSS/forms.css

```diff
@@ -0,0 +1,57 @@
+.forms {
+    background: url(http://www.kinyu-z.net/data/wallpapers/35/823942.jpg) center fixed;
```

Comment:
can this image be downloaded locally to speed up the page load? currently, it will try to download it from the internet on page load, thus it is slower.